pmp-client/src/features/services/ListOfServices.tsx

```diff
@@ -0,0 +1,66 @@
+import useServices from './useServices';
+import {
+    Checkbox,
+    DataTable,
+    DataTableBody,
+    DataTableCell,
+    DataTableContent,
+    DataTableHead,
+    DataTableHeadCell,
+    DataTableRow
+} from 'rmwc';
+import { ChangeEvent } from 'react';
+import useSelectedServices from './useSelectedServices';
+
+const ListofServices = () => {
+    const list = useServices();
```

Comment:
Filter this to only show services in the currently selected environment. see useEnvironment.ts